src/components/common/CardSignInRequired.vue

```diff
@@ -81,6 +86,9 @@ export default {
         : `/create`
     }
   },
+  mounted() {
+    this.currentNetwork = this.networks.find(({ id }) => id === this.network)
```

Comment:
sooooo much code repetition

Reply:
@Bitcoinera here you forgot to replace with getter?

Reply:
yeah, I forgot

Reply:
Fixed ✌️